magnum/drivers/cluster_api/templates/cluster.yaml.jinja2

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: azimuth.stackhpc.com/v1alpha1
+kind: Cluster
+metadata:
+  # TODO: annotations for user_id, etc?
+  name: "{{ cluster.uuid }}"
+spec:
+  addons:
+    apps: true
+    certManager: false
+    dashboard: true
+    ingress: false
+    monitoring: true
+  autohealing: true
+  # TODO: should this not also be created in this file! Maybe with trusts?
+  cloudCredentialsSecretName: "cluster-{{ cluster.uuid }}"
```

Comment:
lets add this in this file, clouds.yaml with the trust in it. See: https://github.com/stackhpc/azimuth/blob/0edf93d1971633bcc8505eb3b89fedbe1a0b04e6/api/azimuth/cluster_api/openstack.py#L34